Products/Zuul/routers/device.py

```diff
@@ -317,7 +317,9 @@ def setInfo(self, **data):
         @rtype: DirectResponse
         """
         facade = self._getFacade()
-        if not Zuul.checkPermission('Manage Device', self.context):
+        if not (Zuul.checkPermission('Manage Device', self.context) or (
```

Comment:
This permission is defined in `Products/ZenModel/ZenossSecurity.py` - why use magic string instead? Same for `"Change Device Production State"`.